contributions/api/controllers/contributions.py

```diff
@@ -823,4 +822,98 @@ def talents_get(token_info=None, id=None, talent_id=None):
         return talent
 
 def ok_search():
-    return rs_handlers.return_200("okay")
\ No newline at end of file
+    return rs_handlers.return_200("okay")
+
+def admin_reviewers_post(token_info):
+    # check if the logged in user is in the reviewers db
```

Comment:
Could you please update this comment and the below method name to clarify that this to check whether the user is an admin? Thanks.

Reply:
Updated